src/lib/route-broadcaster.js

```diff
@@ -121,6 +124,26 @@ class RouteBroadcaster {
       ]
     }
   }
+
+  // Shift the graph down by a small amount so that precision rounding doesn't
+  // cause UnacceptableRateErrors.
+  * _shiftRoute (route) {
```

Comment:
Shifting the points down by a fixed amount should be in `five-bells-routing`.

But `_getScaleAdjustment` can stay here.